python-avocado.spec

```diff
@@ -59,6 +59,9 @@ Source0: https://github.com/avocado-framework/%{srcname}/archive/%{commit}.tar.g
 BuildArch: noarch
 BuildRequires: procps-ng
 BuildRequires: kmod
+BuildRequires: libcdio
```

Comment:
Nice catch! But if these new BRs are just for the benefit of the tests, it would be more appropriate to put them in the `%if %{with_tests}` block below.